### DISCUSSION TITLE
Is there any methods or researches about mathcing single node or subgraph from two different graphs?

### DISCUSSION THREAD

**Discussion by jiaruHithub** [pyg-team/pytorch_geometric#4794] [Q&A] [answered]
Hi, is there any methods or researches about mathcing single node or subgraph from two different graphs? please tell me.
thanks!

**Comment by rusty1s** [comment, marked as answer]
How about:
* https://github.com/rusty1s/deep-graph-matching-consensus
* https://github.com/snap-stanford/neural-subgraph-learning-GNN

**Reply by jiaruHithub** [reply]
Ok, thanks a lot!
